staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -321,6 +321,14 @@ var (
 		"UPDATE",
 		"WATCH",
 		"WATCHLIST")
+
+	//these are the valid connect request which we report in our metrics.
```

Comment:
// These are the valid connect requests which we report in our metrics.

Reply:
Thanks.
Fixed！